.github/PRIVACY_POLICY.md

```diff
@@ -37,11 +37,16 @@ Xeleration
 Houston, TX 77494
 United States
 hello@xel.company
-1 (409) 422-5425
++1 (409) 422-5425
 
 Cameron Samuels
 Katy, TX 77494
 United States
 biz@cameronsamuels.com
+
+Toby Latino
+Prairieville, LA 70769
+United States
++1 (225) 290-4958
```

Comment:
I can create an `@xel.company` email for you and then you won't have to put that phone number. That will be better because I have my email listed and not a phone number. So put `toby@xel.company` instead of your phone number (the email is forward only and won't require login)